### DISCUSSION TITLE
Notification of service usage

### DISCUSSION THREAD

**Discussion by Badatech** [airbytehq/airbyte#40594] [General]
Hi!

Is there any way to get preset notifications about the usage of services, API calls, or any other components included in our package? It would be really helpful for planning costs and maintaining business continuity.

Thanks in advance!

**Comment by nataliekwong** [comment]
Hey @Badatech , if you are looking for a monitoring solution through Airbyte's API, you can list the jobs related to your connections, and then get each job (or sync) status and details, which will provide a sync summary that includes how much data was extracted and loaded. 

Alternatively, you can set up your logs to send to an external S3 bucket, and integrate that with any observability tools in your stack. You can read more about that here: https://docs.airbyte.com/deploying-airbyte/on-kubernetes-via-helm#external-logs-with-s3.

**Reply by Badatech** [reply]
Thank You, I'll check it

**Comment by nataliekwong** [comment]
Great! You can reopen this if you have future questions.
